gooddata-pandas/tests/dataframe/test_dataframe_for_exec_def.py

```diff
@@ -6,7 +6,7 @@
 
 from gooddata_pandas import DataFrameFactory
 from gooddata_sdk import Attribute, ExecutionDefinition, ObjId, SimpleMetric, TotalDefinition, TotalDimension
-from gooddata_sdk.compute.model.execution import ResultSizeLimitsExceeded
+from gooddata_sdk.compute.model.execution import ResultSizeBytesLimitExceeded, ResultSizeDimensionsLimitsExceeded
```

Comment:
If I am not mistaken you added these classes in gooddata-sdk/gooddata_sdk/init.py meaning you can use from gooddata_sdk to import them.

Reply:
fixed